module-4/remove_metadata/remove_metadata.py

```diff
@@ -0,0 +1,46 @@
+import os
+import subprocess
+import argparse
+
+def remove_metadata_exiftool(input_file, output_file):
+    """
+    Removes metadata using ExifTool.
+    """
+    try:
+        command = f'exiftool -all= -o {output_file} {input_file}'
+        subprocess.run(command, shell=True, check=True)
+        print(f"Metadata successfully removed using ExifTool. Output: {output_file}")
+    except subprocess.CalledProcessError as e:
+        print(f"Error removing metadata with ExifTool: {e}")
+
+def remove_metadata_qpdf(input_file, output_file):
+    """
+    Removes metadata using QPDF.
+    """
+    try:
+        command = f'qpdf --linearize --object-streams=disable {input_file} {output_file}'
+        subprocess.run(command, shell=True, check=True)
+        print(f"Metadata successfully removed using QPDF. Output: {output_file}")
+    except subprocess.CalledProcessError as e:
+        print(f"Error removing metadata with QPDF: {e}")
+
+def main():
+    parser = argparse.ArgumentParser(description="Remove metadata from a PDF document.")
+    parser.add_argument('-i', '--input', required=True, help='Path to the input PDF file.')
+    parser.add_argument('-o', '--output', required=True, help='Path to the output PDF file.')
+    parser.add_argument('-t', '--tool', choices=['exiftool', 'qpdf'], default='exiftool',
```

Comment:
in cases like that where we have long lines, just split this over multiple lines to maintain readability in your code.